### ISSUE TITLE
[$40] Login/Register Page SSO links should be able to be navigated to using enter key

### ISSUE THREAD

**Issue by mishacucicea** [topcoder-platform/community-app#3001]
**Steps to Reproduce**
1. Go to Members login:https://accounts.topcoder.com/member. 2. Press the tab key to navigate through the interactive elements on the page. 3. Focus the SSO links (github, facebook, etc) and hit Enter.

**Expected Result**
The links should be navigated to (e.g. the same action as if we click on them) when using the Enter key.

**Actual Result**
Nothing happens when the a link is focused and the Enter key is pressed.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
See #2815 for more details.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096630 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096630 has been updated - it has been assigned to gauravseta.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by ThomasKranitsas** [comment]
[500]: Challenge for the updated issue 3001 is creating, rescheduling this event<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
[500]: Challenge for the updated issue 3001 is creating, rescheduling this event<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096712 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096712 has been updated - it has been assigned to gauravseta.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096712 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by rashmi73** [comment]
@veshu PR for this isse at accounts-app repo.
https://github.com/appirio-tech/accounts-app/pull/271

**Comment by crazyk07** [comment]
@rashmi73  Will you be able to work on this ticket in the next 12 hours?  We have upgraded this ticket to a P0 and will increase price as well.  You have a code review failure that needs to be addressed.  Thank you!

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096712 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096712 has been updated - it has been assigned to sexta13.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@sexta13 I am solving it and it is completed.

**Comment by rashmi73** [comment]
@sexta13 also your issue looks complicated and may have focusing issues of outline.I am raising PR in 2mins

**Comment by sexta13** [comment]
@rashmi73 , true :) I've closed my PR. Just fix the conflict files please.

**Comment by rashmi73** [comment]
@sexta13 I am raising new PR itself to avoid confusions, I did try to work on original PR and commited changes but still conflicts. hence making fresh PR thanks

**Comment by rashmi73** [comment]
@sexta13 @crazyk07 PR is https://github.com/appirio-tech/accounts-app/pull/281

**Comment by Oanh-and-only-Oanh** [comment]
passed on Chrome but failed in Safari. Tabbing in Safari does not focus on Focus the SSO links (github, facebook, etc).

**Comment by lakshmiathreya** [comment]
@Oanh-and-only-Oanh we are focusing on Chrome only. @crazyk07 your thoughts?

**Comment by r0hit-gupta** [comment]
@Oanh-and-only-Oanh @lakshmiathreya @crazyk07 Safari ignores the tab key by default. Have a look at https://stackoverflow.com/questions/1848390/safari-ignoring-tabindex

 @Oanh-and-only-Oanh try holding `option` key and then press `tab` key. It works fine then.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096712<br/><br/>```This is an automated message for crazyk via Topcoder X```
